mailgun/route_handlers.py

```diff
@@ -49,7 +69,7 @@ def handle_mailing_list_email_route(request):
     sender_address = sender.address.lower() if sender else ''
     from_address = from_.address.lower() if from_ else ''
     if settings.NO_REPLY_ADDRESS.lower() in (sender_address, from_address):
-        logger.error(u'Caught a bounce loop, dropping it.  POST data:\n%s\n',
```

Comment:
Is this the straw that broke the camel's back that means we'll be implementing a style guide? :)